packages/components/tag/src/tag.scss

```diff
@@ -1,263 +1,134 @@
 :host {
-  --_button-border: var(--sl-color-tag-subtle-idle-border);
+  --_background: var(--sl-color-tag-subtle-idle-background);
+  --_border-color: var(--sl-color-tag-subtle-idle-border);
   --_border-radius: var(--sl-border-radius-default);
+  --_button-size: var(--sl-size-tag-md);
+  --_color: var(--sl-color-tag-subtle-idle-foreground);
+  --_cursor: pointer;
   --_focus-color: var(--sl-color-focusring-inversed);
   --_focus-outline: var(--_focus-color) solid var(--sl-border-width-focusring-inside);
   --_focus-outline-offset: calc(var(--sl-border-width-focusring-offset) * -1);
   --_font: var(--sl-text-tag-label-md);
-  --_gap: var(--sl-space-tag-gap-md);
   --_padding-block: var(--sl-space-tag-block-md) var(--sl-space-tag-block-md);
   --_padding-inline: var(--sl-space-tag-inline-md) var(--sl-space-tag-inline-md);
 
-  display: block;
-  pointer-events: none;
-}
-
-:host([emphasis='bold']) {
-  --_focus-color: var(--sl-color-focusring-default);
-}
-
-.wrapper {
   align-items: center;
   border-radius: var(--_border-radius);
+  cursor: var(--_cursor);
   display: inline-flex;
-  flex-shrink: 0;
   font: var(--_font);
-  gap: var(--_gap);
   max-inline-size: 100%;
-  padding-block: var(--_padding-block);
-  padding-inline: var(--_padding-inline);
-  pointer-events: auto;
-  transition: background-color 200ms linear;
+  overflow: hidden;
   vertical-align: middle;
-  background: var(--sl-color-tag-subtle-idle-background);
-  color: var(--sl-color-tag-subtle-idle-foreground);
-
-  .remove-button {
-    background: var(--sl-color-tag-subtle-idle-background);
-    color: var(--sl-color-tag-subtle-idle-foreground);
-
-    --_button-border: var(--sl-color-tag-subtle-idle-border);
-  }
-
-  .remove-button:hover {
-    background: var(--sl-color-tag-subtle-hover-background);
-  }
-
-  .remove-button:active {
-    background: var(--sl-color-tag-subtle-close-active-foreground);
-  }
-}
-
-:host([disabled]) .wrapper {
-  background: var(--sl-color-tag-subtle-disabled-background);
-  color: var(--sl-color-tag-subtle-disabled-foreground);
-
-  .remove-button {
-    background: var(--sl-color-tag-subtle-disabled-background);
-    color: var(--sl-color-tag-subtle-disabled-foreground);
-
-    --_button-border: var(--sl-color-tag-subtle-disabled-border);
-  }
 }
 
-:host(:hover:not([disabled])) .wrapper {
-  background: var(--sl-color-tag-subtle-hover-background);
-  color: var(--sl-color-tag-subtle-hover-foreground);
-
-  .remove-button {
-    background: var(--sl-color-tag-subtle-hover-background);
-
-    --_button-border: var(--sl-color-tag-subtle-hover-border);
-  }
+slot:hover,
+slot:hover + button,
+button:hover {
+  --_background: var(--sl-color-tag-subtle-hover-background);
```

Comment:
When the bold tag is focused and I hover it (the tag, not the button in it), the background of the button changes, but it should not when I hover over the tag, not the button. Right now it looks like this:

<img width="181" alt="image" src="https://github.com/user-attachments/assets/4f42c0c6-ab64-45a5-b6ae-902801a9b14e">